_posts/2020-11-23-bisq-v1.5.0-highlights.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Bisq v1.5.0 Released: Highlights"
+author: m52go
+excerpt: "Bisq v1.5.0 was released earlier today, bringing SegWit support across the trading protocol as well as a slew of updates to make trading better and more convenient. <br><br>"
+lang: en
+---
+
+Bisq v1.5.0 was just released, and it's quite a notable update.
+
+## SegWit Integrated Completed
+
+Bisq v1.4.2 introduced segwit addresses for funding and withdrawing, meaning you could send funds from an external wallet to a `bech32` address in Bisq and withdraw funds from Bisq to an external `bech32` address. While this was a huge step forward, trade transactions didn't use segwit, and no Bisq transactions took advantage of fee optimizations enabled by segwit.
```

Comment:
Format: Bech32 looks like a link, but it isn't. I propose to use italic or link this: https://en.bitcoin.it/wiki/Bech32